index.html

```diff
@@ -0,0 +1,129 @@
+                and other applied science.
+            </li>
+            <li>
+                <span>1942</span>- Receives a Ph.D. in Genetics and Plant Pathology
+
+            </li>
+            <li>
+                <span>1944</span>- Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and
+                flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000
+                different strains of disease resistent wheat - including different varieties for each major climate on
+                Earth.
+
+            </li>
+            <li>
+                <span>1945</span>- Discovers a way to grown wheat twice each season, doubling wheat yields
+            </li>
+            <li>
+                <span>1953</span>- crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed,
+                creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+
+            </li>
+            <li>
+                <span>1962</span>- Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent
+                in time to help mitigate mass starvation due to a rapidly expanding population
+
+            </li>
+
+            <li>
+                <span>1970</span>- receives the Nobel Peace Prize
+            </li>
+
+            <li>
+                <span>1983</span>- helps seven African countries dramatically increase their maize and sorghum yields
+            </li>
+            <li>
+                <span>1984</span>- becomes a distinguished professor at Texas A&M University
+            </li>
+            <li>
+                <span>2005</span>- states "we will have to double the world food supply by 2050." Argues that
+                genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says
+                that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals
+                for a long time. Long before we called it science, people were selecting the best breeds."
+            </li>
+            <li>
+                <span>2009</span>- dies at the age of 95.
+            </li>
+        </ul>
+
+        <p class="value1">
+            "Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering
+            intellect, persistence and scientific vision can make to human peace and progress."
+        </p>
+
+        <p class="value">
+            -- Indian Prime Minister Manmohan Singh
+        </p>
+    </div>
+
+    <footer>
+        <h3>
+            If you have time, you should read more about this incredible human being on his <a
+                href="https://en.wikipedia.org/wiki/Norman_Borlaug" id="tribute-link" target="_blank"
+                rel="noopener noreferrer">Wikipedia entry.</a>
+        </h3>
+    </footer>
+
+</body>
+
```

Comment:
Please do not add blank lines or spaces without a reason. Add blank lines to separate large or logical code blocks. 💪